test/PowerShellEditorServices.Test.E2E/DebugAdapterProtocolMessageTests.cs

```diff
@@ -50,6 +53,11 @@ public async Task InitializeAsync()
                 options
                     .WithInput(_psesProcess.OutputStream)
                     .WithOutput(_psesProcess.InputStream)
+                    .ConfigureLogging(builder =>
+                        builder
+                        .AddDebug()
```

Comment:
What does `AddDebug()` do?

Reply:
It adds a log provider to system.diagnostics.trace so you can see anything that's logged in the debug panel. I plan to expand on this a lot more and maybe just replace the host logger entirely with the trace writer that doesn't have any dependencies not already in Powershell anyways. Again that's further scope stuff, this is focused on serial log